test/e2e/servicecatalog/podpreset.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package servicecatalog
```

Comment:
you have to add `_ "k8s.io/kubernetes/test/e2e/servicecatalog"` to test/e2e/e2e_test.go or this test won't be included/run in the test suite